src/app/helptext/system/cloudcredentials.ts

```diff
@@ -87,34 +87,13 @@ export const helptext_system_cloudcredentials = {
   },
 
   account_b2: {
-    placeholder: T("Account ID or Application Key ID"),
-    tooltip: T(
-      'Alphanumeric <a\
- href="https://www.backblaze.com/b2/cloud-storage.html"\
- target="_blank">Backblaze B2</a> ID. Find an Account ID\
- or applicationKeyID by logging in to the account,\
- clicking <i>Buckets</i>, and clicking\
- <i>Show Account ID and Application Key</i>. Enter the\
- <i>Account ID</i> to associate the entire account or\
- generate a new <i>Application Key</i>. The <i>keyID</i>\
- replaces the Account ID and the key string is used in\
- place of the <i>Master Application Key</i>.'
-    )
+    placeholder: T("Key ID"),
+    tooltip: T(''),
```

Comment:
Alphanumeric `<a href="https://www.backblaze.com/b2/cloud-storage.html" target="_blank">Backblaze B2</a>` Application Key ID. To generate a new application key, log in to the Backblaze account, go to the `<i>App Keys</i>` page, and add a new application key. Copy the application `<i>keyID</i>` string to this field.